### DISCUSSION TITLE
I want to make two additional api calls after a successful custom endpoint call?

### DISCUSSION THREAD

**Discussion by edmcquinn** [danny-avila/LibreChat#2164] [Troubleshooting]
I want to make two additional api calls after a successful custom endpoint call that sends the message to get toxicity and factuality estimates on the LLM response. Where is the main place where api calls are made in the code?

**Comment by danny-avila** [comment]
https://github.com/danny-avila/LibreChat/blob/45a95acec2c82ab5358a06014b1ddbf26ee04135/api/app/clients/OpenAIClient.js#L1100
